playbooks/templates/.env.j2

```diff
@@ -227,9 +227,8 @@ SKYNET_ACCOUNTS_PORT=3000
 # REQUIRED: manually defined by user, or default will be used
 BLOCKER_LOG_LEVEL={{ webportal_common_config.blocker_log_level }}
 
-# OPTIONAL: manually defined by user, can be blank
-BLOCKER_NGINX_CACHE_PURGE_LIST={{ webportal_common_config.blocker_nginx_cache_purge_list | default('') }}
-BLOCKER_NGINX_CACHE_PURGE_LOCK={{ webportal_common_config.blocker_nginx_cache_purge_lock | default('') }}
+# OPTIONAL: manually defined by user
+BLOCKER_PORTALS_SYNC={{ webportal_common_config.blocker_portals_sync }}
```

Comment:
If is it optional we should add a default value here so that Ansible doesn't complain when it isn't defined.

Is this a boolean? String?

You can copy the syntax used in the lines you removed to add a default value.

Reply:
Are `BLOCKER_NGINX_...` used somewhere?
Can we safely remove them?
We need to keep backwards compatibility of playbooks if someone (we or portal operators) deploys older/stable versions of the stack.